packages/next-swc/crates/next-core/src/sass.rs

```diff
@@ -37,12 +42,17 @@ pub async fn maybe_add_sass_loader(
             let mut loaders = rule.loaders.await?.clone_value();
             loaders.push(loader);
             rule.loaders = WebpackLoaderItemsVc::cell(loaders);
+            if let Some(rename_as) = rule.rename_as.as_mut() {
+                rename_as.push_str(rename);
```

Comment:
Can you explain this, I don't understand why we would append to an already established rename rule? Should this be an error case?

Reply:
yeah, I guess we should not apply our sass logic when the `rename_as` is used.

e. g. they could

```
// apply their own sass transform
"*.sass": {
  loader: ["sass-loader"],
  as: "*.css"
}
```

```
// convert to js
"*.sass": {
  loader: ["sass-to-js-loader"],
  as: "*.js"
}
```

Reply:
I'll fix it in a separate PR